_layouts/default2.html

```diff
@@ -41,5 +41,7 @@ <h3 class="masthead-title">
       })(document);
     </script>
         <script src="{{ site.baseurl }}/js/main.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
Why do we need so many 'default' layouts? (_layouts/default2.html, default3.html, default4.html) What is the benefit of this redundancy? If you need more layouts, can't we rename these to something more "meaningful"???

Reply:
You already added a local jquery file in `js/`. Do we still need to fetch jquery from CDN??

Reply:
Also, both these jquery files have different versions. The one we were getting from CDN is 2.1.3 and the one you just added locally is 1.11.2. Are you specifically looking for a feature in older version that's not present in newer one?

Reply:
Hey @Nalinc 
1) The live instance is hosted using a bot account. Here's the [link](https://sachin10101998bot.github.io/aima-exercises/). The bot account is used because the master branch has been used for building the answer submission feature and for fetching Github Contents API. It becomes difficult to track all those places and change them from master to some other branch (say gh-pages). Therefore the live instance is hosted using the main branch of bot account, where the only changes I had to make while pushing it to the main repo of Aimacode were replacing sachin10101998bot with Aimacode at required places. Moreover, sample answer files can be added in the bot account to check how the answers look as in Exercise 1.1.
2) Jekyll paginate came with Lanyon theme but since we aren't using pagination in our project so I have removed it from `_config.yaml`
3) Names of layout files have been updated to make much more sense and unnecessary layouts have been removed.
4) Local Jquery file has been removed. It's being fetched from CDN now. It was not needed in this PR.